Scripts/WikiUpdater.cs

```diff
@@ -0,0 +1,379 @@
+
+        var organellePages = new List<TranslationPair>();
+
+        foreach (var organelle in organelles)
+        {
+            var name = organelle.TextContent.Trim();
+            var url = $"https://wiki.revolutionarygamesstudio.com/wiki/{name.Replace(" ", "_")}";
+
+            var untranslatedOrganelleName = name.ToUpperInvariant().Replace(" ", "_");
+
+            ColourConsole.WriteSuccessLine($"Found organelle {name}");
+
+            var body = (await HtmlReader.RetrieveHtmlDocument(url, cancellationToken)).Body!;
+
+            // Get the internal name for cross-referencing against in-game data for the organelle
+            var internalName = body.QuerySelector("#info-box-internal-name")!.TextContent.Trim();
+
+            var sections = GetMainBodySections(body);
+            var untranslatedSections = sections.Select(
+                section => UntranslateSection(section, untranslatedOrganelleName)).ToList();
+
+            var untranslatedPage = new Wiki.Page(
+                $"WIKI_PAGE_{untranslatedOrganelleName}",
+                internalName,
+                url,
+                untranslatedSections);
+            var translatedPage = new Wiki.Page(
+                name,
+                internalName,
+                url,
+                sections);
+
+            organellePages.Add(new TranslationPair(untranslatedPage, translatedPage));
+
+            ColourConsole.WriteSuccessLine($"Populated content for organelle with internal name {internalName}");
+        }
+
+        return organellePages;
+    }
+
+    /// <summary>
+    ///   Extracts page sections from the main article body and converts to BBCode. Sections are delineated by h2 tags,
+    ///   which are taken as the headings (or null for the first section).
+    /// </summary>
+    /// <param name="body">Body content of the whole page</param>
+    private static List<Wiki.Page.Section> GetMainBodySections(IHtmlElement body)
+    {
+        var sections = new List<Wiki.Page.Section> { new(null, string.Empty) };
+
+        var children = body.QuerySelector(".mw-parser-output")!.Children;
+        foreach (var child in children)
+        {
+            if (child.TagName == "H2")
+            {
+                // Complete the previous section and start a new one with this heading
+                sections.Add(new Wiki.Page.Section(child.TextContent, string.Empty));
+                continue;
+            }
+
+            string text;
+            switch (child.TagName)
+            {
+                case "P":
+                    text = ConvertTextToBbcode(child.InnerHtml) + "\n\n";
+                    break;
+                case "UL":
+
+                    // Godot 3 does not support lists in BBCode, so use custom formatting
+                    text = child.Children
+                        .Where(c => c.TagName == "LI")
+                        .Select(li => $"[indent]•   {ConvertTextToBbcode(li.InnerHtml)}[/indent]")
```

Comment:
I added a bullet item translation for the news feed. I think it would be good to use that here for consistency (or to reduce the load on the translators. I've seen at least French using a separate bullet list item character than what's the default for English).

Reply:
I'm not sure I see an obvious way to do that here, since this is the English text, not something I could put a translation key in. Maybe sections could be made a format string with the localised bullet character inserted at runtime. That would open a can of worms in regards to whether I should do the same for other characters.

Reply:
I think you should strive to have the text held inside a localized string builder for easy update on language change. That allows easily embedding the bullet point translation followed by the content. I think for now skipping allowing translation special characters that aren't already translated is fine. We can react after the fact to people wanting to translate more text structure elements.

Reply:
One extra point I've just thought of in relation to this: the reason I needed to add the bullet character is that BBCode in Godot 3 doesn't support lists. BBCode in Godot 4 does, so if and when the upgrade happens, ideally we'd replace this with `[li]` elements that Godot 4 presumably has a way to translate.

Not sure I follow your suggestion here. Where would the string builder be?

Reply:
Oh right, this is in the script. Not in the game yet so user language can't be known yet.

I think to be consistent with the news feed, this should just use the em-dash or whatever "—" is called. That uses this translation: https://translate.revolutionarygamesstudio.com/translate/thrive/thrive-game/en/?checksum=235288f133b71457 but as this is running before that, it cannot be easily done.

So my suggestion for now is to just replace the `•   ` part with `—` and let other language translators pick a different character if they want.

Reply:
Updated to an em dash.